addons/hr_holidays/static/src/leave_stats/leave_stats.xml

```diff
@@ -10,7 +10,12 @@
             </div>
             <div t-foreach="state.leaves" t-as="leave" t-key="leave_index" class="d-flex flex-row justify-content-between">
                 <span t-esc="leave.holiday_status_id[1]"/>
-                <span><t t-esc="leave.number_of_days"/> day(s)</span>
+                <t t-if="leave.leave_type_request_unit?.includes('hour')">
+                    <span><t t-esc="leave.number_of_hours"/> hour<t t-if="leave.number_of_hours &gt; 1">s</t></span>
```

Comment:
That should stay as it was before; sadly [plural rules vary a lot](<https://udn.realityripple.com/docs/Mozilla/Localization/Localization_and_Plurals#List_of_Plural_Rules>) depending on the language, and formatting text like this will cause issues for the translation team. Same comment for the other similar changes in the file :)